jdbc_bridge/src/main/java/com/starrocks/jdbcbridge/JDBCBridge.java

```diff
@@ -0,0 +1,27 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.jdbcbridge;
+
+import java.io.File;
+import java.net.URL;
+import java.net.URLClassLoader;
+
+
+
+public class JDBCBridge {
```

Comment:
should comment why need JDBCBridge class.

Reply:
> should comment why need JDBCBridge class.

Added, plz review again.